### ISSUE TITLE
`DeTeXify` option?

### ISSUE THREAD

**Issue by pihart** [cjquines/qboard#188]
Allow people to perform a detexify (or more generally, mathpix) query against a selection of objects?

The general solution of using mathpix seems extremely useful for converting both notes and handwritten lecture slides into tex, while the detexify-only solution seems useful for when someone is doing this conversion manually and doesn't know the latex for a given symbol. Admittedly, the detexify-only use case is mostly solved by people just using the actual detexify app, though that's necessarily not a perfect solution since users must redraw figures.

(The detexify api is public at https://detexify.kirelabs.org/api/classify)

**Comment by cjquines** [comment]
no

**Comment by cjquines** [comment]
bad

**Comment by cjquines** [comment]
this is blatant feature creep and i will not have it no matter what explanations you give

**Comment by pihart** [comment]
Agreed this is feature creep.

I find it extremely useful for these two specific cases:
- you have lecture slides where you did some working out and you want to convert these to TeX - this is a very common use case among people i work with
- you don't know *how* to format something in LaTeX. So you use the latex feature to add the components (and hand draw symbols, whatever), you move/resize them as you see fit, and then you use the mathpix feature to get a very close LaTeX approximation (this works extraordinarily well)

These cases are difficult to approximate using existing solutions; they both require you to screenshot the board page-by page (or use the to-come image export feature) and also use the unfriendly mathpix UI, but i understand if you don't want this added to qboard.

**Comment by cjquines** [comment]
> this is a very common use case among people i work with

seriously!?!? what?!?!

**Comment by pihart** [comment]
> > this is a very common use case among people i work with
> 
> seriously!?!? what?!?! 

Yes, especially when we use qboard for personal-ish notes. (This is not about proper lectures, per se.)

---

Aside from the value of transcribing work from lecture/tutoring/etc., this feature would give a quite efficient handwriting to TeX workflow. I don't often want to do such a thing in a case where i don't already have the handwriting (via lecture, say), but others may. I'm thinking especially for PROMYS - if people can just write as they would on paper and get both an editable qboard file and TeX source, this is perfect.

^ this is repetitive but above case also works again with the lecture/tutoring case of people working together on, say, a research lab
